app/bundles/CoreBundle/Entity/TranslationEntityInterface.php

```diff
@@ -12,7 +12,7 @@ interface TranslationEntityInterface
     /**
      * Get translation parent.
      *
-     * @return TranslationEntityInterface
+     * @return mixed
```

Comment:
I cant find any instances where this isn't `TranslationEntityInterface` or `null`.

Reply:
If it can be `TranslationEntityInterface |null` I'd suggest we do that instead and update the related methods/props with the same type.

Reply:
cc @mabumusa1

Reply:
@diaboloshogunate we decided to use `mixed` because this trait is used in multiple contexts and `phpstan` can not handle this, you can read more about this [issue here](https://github.com/phpstan/phpstan/discussions/9265)